### PR TITLE
Fixed all warnings that GHC gave me.

### PR DESCRIPTION
**Pull request by Rembane**
But there are probably more we want to add. So this is not done yet! :D 
https://downloads.haskell.org/~ghc/8.6.3/docs/html/users_guide/using-warnings.html#options-sanity

### CODE REVIEW

**Comment by Rembane** [comment]
According to me it's ready to merge. Please have a review! :D

File: src/Model.hs [hunk 1]
```diff
@@ -92,15 +94,14 @@ update
 update = do
   c       <- asks ccCfg
   dateNow <- liftIO $ fmap (view _zonedTimeToLocalTime) getZonedTime
-  let (textday, date) =
+  let (textday, d) =
```
Comment (adamse) [inline src/Model.hs:97]: I don't think there is a warning for too long variable names ... Hmmm :)

**Review by adamse — APPROVED** [review]
Oki!

File: src/Model.hs (hunk 1 above)
Comment (Rembane) [reply src/Model.hs:97]: Sadly there isn't. :( 
On a more serious note, GHC gave warnings for `date` shadowing some function from Thyme so I just renamed `date`.

**Pull request merged**